### DISCUSSION TITLE
Backup  quota exceeded

### DISCUSSION THREAD

**Discussion by Maxwellslaw** [seedvault-app/seedvault#200] [Q&A]
Hi all,

Thanks first for the great app.

I have a question regarding the message "Backup quota exceeded" for some apps i have. 
How are they being handled? Is the user data still saved or are they not touched at all? 
I alos get this info when i mark only to save the app-data!

Is there a way to increase the size per app?

Thanks and best regards
Joerg

**Comment by beerisgood** [comment]
Related:
https://github.com/seedvault-app/seedvault/issues/183

**Reply by grote** [reply]
Thanks, I think this answers the question: https://github.com/seedvault-app/seedvault/issues/183#issuecomment-748931399

**Reply by Maxwellslaw** [reply]
Sadly 😄 Yes. 
Thanks for the quick reply
Joerg
